### PR TITLE
SED-2053 add explicit execution result column in executions list view

### PR DESCRIPTION
**Pull request by neogucky**

### CODE REVIEW

File: projects/step-frontend/src/lib/modules/execution/components/execution-result/execution-result.component.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<span [matTooltip]="displayResult" [class]="statusClass">
-  <step-icon [name]="icon" [class.spin-animation]="icon === 'refresh-cw'">
-  </step-icon>
+<span [matTooltip]="displayResult" [ngClass]="[statusClass, 'step-icon-' + result]">
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/execution/components/execution-result/execution-result.component.html:1]: What is the reason of this change, if `statusClass` itself will contain 'step-icon-' + result.
Check line 33 in component's code

Comment (neogucky) [reply projects/step-frontend/src/lib/modules/execution/components/execution-result/execution-result.component.html]: Actually it contained ".step-icon-" + result that's why it did not work and I did not think it might actually contain what I needed. Fixed it now!

**Review by bojidaryovchev — APPROVED** [review]
